README.md

```diff
@@ -129,12 +143,14 @@ network {
   mode = "bridge"
 }
 ```
-You need to install CNI plugins on nomad client nodes under `/opt/cni/bin` before you can use `bridge` networks.
+You need to install CNI plugins on Nomad client nodes under `/opt/cni/bin` before you can use `bridge` networks.
```

Comment:
This option is configurable in 0.12.X+, but this is the default location

Reply:
I was not sure if the CNI plugins are installed by default. Added the instructions just to be explicit.